mirror/modules/sink/http.go

```diff
@@ -67,6 +68,18 @@ func NewHTTP(ctx *mirror.ModuleContext, cfg []byte) (mirror.Module, error) {
 		return nil, err
 	}
 
+	followRedirect, err := strconv.ParseBool(c.FollowRedirect)
+	if err != nil {
+		return nil, fmt.Errorf("follow_redirect: %w", err)
+	}
+
+	checkRedirect := (func(req *http.Request, via []*http.Request) error)(nil)
```

Comment:
Would be cleaner to 

1. create the http client (move from HTTP struct creation)
2. if !followRedirect then set the call back
3. use http client var in HTTP struct creation

Btw your condition should be reversed, set the callback if followRedirect is false

Reply:
Thanks for the advice, it's indeed much more readable.